libs/hdf-converters/test/mappers/forward/dbprotect_mapper.spec.ts

```diff
@@ -0,0 +1,121 @@
+import fs from 'fs';
+import {DBProtectMapper} from '../../../src/dbprotect-mapper';
+import {omitVersions} from '../../utils';
+
+describe('dbprotect_mapper_check', () => {
+  it('Successfully converts DBProtect check scan targeted at a local/cloned repository data', () => {
+    const mapper = new DBProtectMapper(
+      fs.readFileSync(
+        'sample_jsons/dbprotect_mapper/sample_input_report/DbProtect-Check-Results-Details-XML-Sample.xml',
+        {encoding: 'utf-8'}
+      )
+    );
+
+    // fs.writeFileSync(
```

Comment:
Would it make SonarQube happy to just have this once at the top of the file instead of leaving each one commented?

Reply:
Yeah SQ complains about basically any commented out code since it (more or less rightfully) thinks that you can just remove it and if you want it back then grab it from source control.  However, we use it frequently enough that it's just easier to have on hand always so I'm fine with this being a WONTFIX.